react-props-and-expressions/src/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
```

Comment:
Missed it in the previous ones but you need to add `/client` to the ReactDOM import:
```JAVASCRIPT
import ReactDOM from 'react-dom/client';
```